release-managers.md

```diff
@@ -126,7 +126,6 @@ referred to as Release Manager shadows. They are responsible for:
 GitHub Mentions: @kubernetes/release-engineering
 
 - Arnaud Meukam ([@ameukam](https://github.com/ameukam))
-- Gianluca Arbezzano ([@gianarb](https://github.com/gianarb))
```

Comment:
we'll need an offboarding issue here to make sure we get all the steps covered 👍